data/json/mapgen/farm_dairy.json

```diff
@@ -572,6 +572,8 @@
         "b": "t_floor",
         "d": "t_floor",
         "D": "t_floor",
+        "K": "t_floor",
+        "L": "t_fence_post",
```

Comment:
These symbols are simply meant to mark the position where the roof is located, and the roof is actually created at ground floor. There's no need in any change.

Reply:
Will make them `t_open_air`.